src/Tracker/utils/index.js

```diff
@@ -0,0 +1,45 @@
+export const sendAnalytics = (payload) => {
+  const request = new XMLHttpRequest()
+  request.open('POST', 'https://analytics-sdk-dev.onfido.com/v1/import')
+  request.setRequestHeader('Content-Type', "application/json")
+  request.setRequestHeader('Authorization', "Basic M002Z3tQWURwWTlUanlDKDNRTHIqTDZOQ1hVY2FLZUU6")
```

Comment:
Shouldn't this be hidden from the public repo?

Reply:
It's not a real secret apparently, I've hardcoded values inside this PoC but I will move then in the webpack config file when this is ready

Reply:
Agree that this is not a real secret. Everyone using the JS SDK will be able to see it